examples/examples_with_threads/test_listenning_agent.metta

```diff
@@ -1,22 +1,27 @@
-! (import! &self motto)
-! (bind! &a1 (listening-agent "examples/examples_with_threads/simple_call.msa"))
-! (&a1 (user "hi"))
-! (&a1 .set_event "speech" (py-dict (("text" "who is the 6 president of France"))))
+!(import! &self motto)
+!(import! &self agents)
+!(import! &self agents:events)
+!(bind! &events (basic-event-bus))
+
+!(bind! &a1 (listening-agent "examples/examples_with_threads/simple_call.msa"  &events))
+!(&a1 .start)
+!(publish-event &events "speech" (py-dict (("text" "who is the 6 president of France"))))
 ! ((py-atom time.sleep) 5)
 ! (&a1 .say)
-! (&a1 .set_event "speech" (py-dict (("text" "who is John Legend? tell me many sentences"))))
-! ((py-atom time.sleep) 0.7)
+! (publish-event &events "speech" (py-dict (("text" "who is John Legend? tell me many sentences"))))
```

Comment:
A minor question: why in simple_call.msa there is
`!(queue-subscription speech handle-speech)`
with `speech` as a symbol, while here we have `"speech"` as a string? I'd prefer having both of the same type if it works